### ISSUE TITLE
unify preprocess layer in model zoo

### ISSUE THREAD

**Issue by QiJune** [sql-machine-learning/elasticdl#1904]
I find that there is some keras preprocessing layer in [census_wide_deep_model](https://github.com/sql-machine-learning/elasticdl/blob/develop/model_zoo/census_wide_deep_model/keras_process_layer.py). Could we move these preprocessing layer into elasticdl preprocessing directory? Or if the preprocessing layer has been implemented already, we could import it from elasticdl preprocessing library.

**Comment by workingloong** [comment]
Yes, we can import from elasticdl_preprocessing in `census_wide_deep_model`.
